templates/types/streaming/nextjs/app/components/ui/chat/chat-input.tsx

```diff
@@ -113,7 +113,17 @@ export default function ChatInput(
         />
         {process.env.NEXT_PUBLIC_USE_LLAMACLOUD === "true" &&
           props.setRequestData && (
-            <LlamaCloudSelector setRequestData={props.setRequestData} />
```

Comment:
How about we keep exactly this usage and just add the onselect for chat llamaindex as an alternative (remember we want to make it easy to delete this selector)